### DISCUSSION TITLE
I'm in a team without knowing about it and now I can't access my domains.

### DISCUSSION THREAD

**Discussion by davegomez** [vercel/community#6295] [Account Issue] [answered]
### Question

I think I created a team and I don't remember, but I'm in it when I log into my account.

The projects I have hosted are inside that team, but all my domains disappeared and I can't access them anymore, neither I can delete the team.

**Comment by davegomez** [comment]
I already found the Domains under Account Settings -> Domains, but I can't make modifications to one of my domains.

It literally says "You don't have permission to create the domain record."

**Reply by davegomez** [reply]
There is definitely something wrong with my account.

I don't remember creating that team before, and I can't even see even less modify the DNS records of any of my domains.

**Comment by amyegan** [comment]
Hey @davegomez. It sounds like you're on a free team because your account was recently updated to our new structure. You can read more about the change here: https://vercel.com/changelog/2024-01-account-changes

If you go to your account **Settings** and click into the **Domains** tab there, do you see all of your domains listed?

| Account Settings                           | Domains                            |
| ----------------------------------- | ----------------------------------- | 
|<img width="416" alt="Screenshot" src="https://github.com/vercel/community/assets/8247105/b3b31aa1-59e9-477a-bb98-f34a3f1e53c3" style="display:inline">|<img width="431" alt="Screenshot" src="https://github.com/vercel/community/assets/8247105/35ae17bf-b541-40b3-8772-c92cfd6502da" style="display:inline">|

**Reply by davegomez** [reply]
Yes @amyegan I can see the list of domains, but I can't make any modification neither see the already defined DNS records, basically they appear empty.

![Screenshot 2024-03-27 at 18 25 31](https://github.com/vercel/community/assets/282903/7226a83d-1006-414c-9144-ca9b32a8d864)

**Reply by amyegan** [reply, marked as answer]
Are you able to move the domain to your new hobby team?

<img width="1232" alt="Screenshot" src="https://github.com/vercel/community/assets/8247105/3e61105c-58a8-48b2-990c-b21b126761a6">

The TXT record may no longer be needed after you move the domain to your team.

**Reply by davegomez** [reply]
Yes @amyegan, moving the domain to my hobby team worked. Thank you.
